cloud/cmd/cloudcore/app/options/options.go

```diff
@@ -24,7 +24,7 @@ import (
 	cliflag "k8s.io/component-base/cli/flag"
 
 	"github.com/kubeedge/kubeedge/common/constants"
-	config "github.com/kubeedge/kubeedge/pkg/apis/cloudcore/v1alpha1"
+	config "github.com/kubeedge/kubeedge/pkg/apis/componentconfigs/cloudcore/v1alpha1"
```

Comment:
Use singular instead of plural here? When metioning the API, is ComponentConfig API (singular)

Reply:
Done.